src/Normalizer/FieldNormalizer.php

```diff
@@ -96,7 +96,7 @@ public function denormalize($data, $class, $format = NULL, array $context = arra
    * @return array
    *   The array of normalized field items.
    */
-  protected function normalizeFieldItems($field, $format, $context) {
+  protected function normalizeFieldItems(FieldItemListInterface $field, string $format, array $context) {
```

Comment:
Ok, will go for that and see how can I avoid `phpcs` complaining about those, I really forget about Php 5 sometimes. Thanks!